app/src/main/AndroidManifest.xml

```diff
@@ -20,6 +20,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".presentation.view.activity.HomeActivity"></activity>
+        <activity android:name=".presentation.view.activity.HomeActivity"
+            android:configChanges="keyboard|keyboardHidden|orientation|screenLayout|uiMode|screenSize|smallestScreenSize"></activity>
```

Comment:
No, I read better and only need orientation and screenSize, I will make a commit to fix that.